packages/runtime-common/tests/query-test.ts

```diff
@@ -2463,6 +2440,72 @@ const tests = Object.freeze({
       `'null' is not a permitted value in a 'range' filter`,
     );
   },
+  'can get prerendered cards (html + css) from the indexer': async (
+    assert,
+    { indexer, loader, testCards },
+  ) => {
+    let { vangogh } = testCards;
+    await setupIndex(indexer, [
+      {
+        card: vangogh,
+        data: {
+          embedded_html: {
+            default: '<div>Van Gogh</div',
+          },
+          deps: [`${testRealmURL}person`],
+          realm_version: 2, // 2 because we're adding modules to the index after setup
```

Comment:
I'm not entirely sure this is the correct way to setup this test. Since `module` and `css` is added to the index in the next lines (using `updateEntry`), those will have `realm_version` 2, and we need card instances to match that because the prerendered card query needs the versions to match.